examples.py

```diff
@@ -1,32 +1,62 @@
+#   -*- coding: utf-8 -*-
+#
+#   This file is part of SKALE Oracle-demo
+#
+#   Copyright (C) 2019 SKALE Labs
+#
+#   This program is free software: you can redistribute it and/or modify
+#   it under the terms of the GNU Affero General Public License as published by
+#   the Free Software Foundation, either version 3 of the License, or
+#   (at your option) any later version.
+#
+#   This program is distributed in the hope that it will be useful,
+#   but WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#   GNU Affero General Public License for more details.
+#
+#   You should have received a copy of the GNU Affero General Public License
+#   along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
 import time
-from Crypto.Hash import keccak
 import json
 import requests
+import os
+from web3 import Web3
+from Crypto.Hash import keccak
 
 MIN_POW_RESULT = 10000
 MAX_POW_NUMBER = 100000
-url='http://127.0.0.1:2234'
+
+url = os.environ.get('ENDPOINT')
```

Comment:
If no ENDPOINT provided default value should be used (127.0.0.1...)

Reply:
Resolved